internal/cmd/cat/cat.go

```diff
@@ -2,29 +2,58 @@ package cat
 
 import (
 	"fmt"
-
 	"github.com/aws-cloudformation/rain/cft/format"
-	"github.com/aws-cloudformation/rain/internal/ui"
-
 	"github.com/aws-cloudformation/rain/cft/parse"
 	"github.com/aws-cloudformation/rain/internal/aws/cfn"
 	"github.com/aws-cloudformation/rain/internal/console/spinner"
+	"github.com/aws-cloudformation/rain/internal/dc"
+	"github.com/aws-cloudformation/rain/internal/ui"
 	"github.com/spf13/cobra"
 )
 
 var transformed = false
 var unformatted = false
+var config = false
 
 // Cmd is the cat command's entrypoint
 var Cmd = &cobra.Command{
-	Use:                   "cat <stack>",
-	Short:                 "Get the CloudFormation template from a running stack",
-	Long:                  "Downloads the template used to deploy <stack> and prints it to stdout.",
+	Use:   "cat <stack>",
+	Short: "Get the CloudFormation template from a running stack",
+	Long: `Downloads the template or the configuration file used to deploy <stack> and prints it to stdout.
+
+The config flag can be used to get the rain config file for the stack instead of the template.
```

Comment:
The generated markdown already includes an example. I suggest removing the Example section and changing "The config flag" to "The `--config` flag"

Reply:
changes implemented with the last commit.